airgun/views/host_new.py

```diff
@@ -513,20 +513,30 @@ class roles(Tab):
         class variables(Tab):
             TAB_NAME = 'Variables'
             ROOT = './/div[@class="ansible-host-detail"]'
+
+            actions = Button(locator='//tbody/tr/td[7]//button')
+            delete = Button(locator='//button[@role="menuitem"]')
+            confirm = Button(locator='//button[@data-ouia-component-id="btn-modal-confirm"]')
             table = Table(
                 locator='.//table[contains(@class, "pf-c-table")]',
                 column_widgets={
                     'Name': Text('.//a'),
                     'Ansible role': Text('./span'),
                     'Type': Text('./span'),
                     # the next field can also be a form group
-                    'Value': Text('./span'),
+                    'Value': TextInput(locator='//textarea[contains(@class, "pf-c-form")]'),
                     'Source attribute': Text('./span'),
                     # The next 2 buttons are hidden by default, but appear in this order
-                    5: Button(locator='.//button[@aria-label="Cancel editing override button"]'),
-                    6: Button(locator='.//button[@aria-label="Submit override button"]'),
+                    6: Button(locator='.//button[@aria-label="Cancel editing override button"]'),
+                    7: Button(locator='.//button[@aria-label="Submit override button"]'),
                     # Clicking this button hides it, and displays the previous 2
-                    7: Button(locator='.//button[@aria-label="Edit override button"]'),
+                    5: Button(locator='.//button[@aria-label="Edit override button"]'),
```

Comment:
Why changing the buttons order here? 
and could you add some descriptions about change to this locators in the description?

Reply:
Locators 6 and 7 are not supported, and there is no column for the edit, submit, or cancel buttons. Based on the index, these buttons correspond to key 5. To enable editing, I added the edit locator under key 5. For submitting the value, I created another variable with the same key (5) but for the submit action. However, combining them does not work—I tried that as well.